docs/antrea-network-policy.md

```diff
@@ -601,6 +601,9 @@ order in which they are enforced.
 A ClusterGroup (CG) CRD is a specification of how workloads are grouped together.
 It allows admins to group Pods using traditional label selectors, which can then
 be referenced in ACNP in place of stand-alone `podSelector` and/or `namespaceSelector`.
+In addition, ClusterGroup also supports Pod grouping by `serviceReference`. ClusterGroup
```

Comment:
maybe use `serviceReference` with backticks consistently in the doc?

Reply:
Done